go/vt/vtgate/evalengine/expressions.go

```diff
@@ -67,14 +69,41 @@ type (
 		Offset    int
 		Collation collations.ID
 	}
+	Tuple []Expr
 )
 
+func (t Tuple) Evaluate(env ExpressionEnv) (EvalResult, error) {
```

Comment:
don't we want comments on public methods? these are all interface implementations, so I don't personally miss them, but the linter should have complained, no?

Reply:
For some reason it didn't complain. I'll fix this